bundles/at.medevit.elexis.agenda.ui/rsc/html/switchParallel.html

```diff
@@ -17,7 +17,7 @@
     <script src="../fullcalendar.js"></script>
     <script src="../locale-all.js"></script>
     <script src="../scheduler.min.js"></script>
-
+	<script src="https://html2canvas.hertzen.com/dist/html2canvas.min.js"></script>

```

Comment:
Bitte die html2canvas.min.js Datei laden und zu den anderen js Dateien in das rsc/lib Verzeichnis ablegen. Ansonsten funktioniert das nicht mehr sollte diese url nicht mehr erreichbar sein.